run/helm/snap/charts/influxdb/values.yaml

```diff
@@ -0,0 +1,63 @@
+## influxdb image version
+## ref: https://hub.docker.com/r/library/influxdb/tags/
+image:
+  repo: "influxdb"
+  tag: "1.2.0"
+  pullPolicy: IfNotPresent
+
+## Port number for influxDB queries and commands
+##
+port_number: 8086
+
+## Persist data to a persitent volume
+##
+persistence:
+  enabled: false
+    ## If defined, volume.beta.kubernetes.io/storage-class: <storageClass>
+    ## Default: volume.alpha.kubernetes.io/storage-class: default
+    ##
+    # storageClass:
+    # size: 8Gi
+
+## Create default user through Kubernetes job
+## Defaults indicated below
+##
+setDefaultUser:
+  enabled: true
+
+  ## Image of the container used for job
+  ## Default: appropriate/curl:latest
+  ##
+  image: appropriate/curl:latest
+
+  ## Deadline for job so it does not retry forever.
+  ## Default: activeDeadline: 300
+  ##
+  activeDeadline: 300
+
+  ## Restart policy for job
+  ## Default: OnFailure
+  restartPolicy: OnFailure
+
+  user:
+    ## The user name
+    ## Default: "admin"
+    username: "root"
+
+    ## User password
+    ## Default: Randomly generated 10 characters of Ascii
+    password: "root"
+
+    ## User privileges
+    ## Default: "WITH ALL PRIVILEGES"
+    privileges: "WITH ALL PRIVILEGES"
+
+## Configure resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+resources:
```

Comment:
We actually found that we needed to up these resource constraints.  InfluxDB recommends about 2Gi for memory